src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.style.display = 'flex';
+spider.style.position = 'static';
+spider.style.margin = 'auto';
```

Comment:
It's boring solution, let's use `clientWidth` and so on

Reply:
There is too many possible solutions😅